### DISCUSSION TITLE
SenseCAP LoRaWAN Starter Kit Getting Started | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#695] [Q&A]
# SenseCAP LoRaWAN Starter Kit Getting Started | Seeed Studio Wiki

SenseCAP LoRaWAN Starter Kit Getting Started

https://wiki.seeedstudio.com/M2_Kit_Getting_Started/

**Comment by korsehindi** [comment]
How to find the source code of this project (XIAO firmware)?

**Reply by john-nistler** [reply]
I would also like to have access to the source code, as I have overwritten it with the blink example, and would like to restore my starter kit to its original shipped state.

**Reply by idreamsi** [reply]
Even the stock firmware binary file can help.

**Reply by Love4yzp** [reply]
You can find the source code at this GitHub repository: [Seeed-Solution/lorawan-kit](https://github.com/Seeed-Solution/lorawan-kit). The file structure of this repository is similar to that of the [SenseCraft-Wio](https://github.com/Seeed-Studio/SenseCraft-Wio) project by Seeed Studio.

**Comment by electrolabopen** [comment]
Hello, how to How can I connect the endpoint of this kit to The Things Networ?  xiao firmware?

**Comment by electrolabopen** [comment]
how can I do to load appkey?
